package.json

```diff
@@ -37,34 +39,32 @@
     "@babel/core": "^7.24.7",
     "@babel/eslint-parser": "^7.19.1",
     "@babel/node": "^7.22.5",
+    "@babel/plugin-transform-react-jsx": "^7.20.7",
     "@babel/preset-env": "^7.19.4",
     "@babel/preset-flow": "^7.18.6",
-    "babel-jest": "^29.2.1",
+    "@babel/register": "^7.24.6",
+    "babel-plugin-module-resolver": "^5.0.2",
     "commander": "^9.4.1",
-    "eslint": "^8.26.0",
-    "eslint-config-airbnb-base": "^15.0.0",
+    "deep-equal-json": "^1.0.0",
+    "eslint": "^8.57.0",
     "eslint-plugin-flowtype": "^8.0.3",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-jest": "^27.1.3",
     "expect": "^29.2.1",
     "flow-bin": "^0.191.0",
-    "jest": "^29.2.1",
     "minimist": "^1.2.7",
-    "rimraf": "^3.0.2"
-  },
-  "jest": {
-    "coverageReporters": [
-      "lcov"
-    ],
-    "coverageDirectory": "reports",
-    "roots": [
-      "<rootDir>/__tests__"
-    ]
+    "mock-property": "^1.0.3",
+    "nyc": "^10.3.2",
+    "object-inspect": "^1.13.2",
+    "object.values": "^1.2.0",
+    "rimraf": "^3.0.2",
+    "tape": "^5.8.1"
   },
   "browserslist": [
     ">0.2%",
     "not dead",
     "not op_mini all",
     "ie 11"
-  ]
+  ],
+  "engines": {
+    "node": ">= 0.4"
+  }
```

Comment:
During the last 2 years Node.js 0.4 has been downloaded a measly 9800 times of 2 694 251 468 total downloads, making only a 0,0004% version usage.

Can you name a single developer that would benefit from this change? Like really, using aria-query and for some reason running an ancient, unsupported Node.js version in production. Probably the only person benefiting would be yourself, you seem to be moving all CI and other from widely used repositories to your own hands. If you weren't a well known developer I wouldn't be suprised if we had another XZ case on our hands. Or maybe the best way to do something like that is be so well known that no one would suspect you :upside_down_face: